src/encoded/schemas/mixins.json

```diff
@@ -239,6 +239,14 @@
             }
         }
     },
+    "external_id": {
```

Comment:
Curious as to why you did not use the already existing dbxrefs mixin?  There is a mechanism in fourfront via namespaces.json and some other magic that allows some dbxrefs to become links - that might be leveraged for some cases?